jis-api/src/main/java/pl/edu/uj/JImageStream/api/core/Filter.java

```diff
@@ -60,9 +61,14 @@ protected int getSourceWidth() {
     }
 
     public void setUp() {
+        startTime = System.currentTimeMillis();
+        logger.info("starting filter");
     }
 
     public void tearDown() {
+
+        logger.info("ending filter");
```

Comment:
fyi, there's a way to write logs without concatenating string. you could use 
logger.info("filter {} has ended, time {} ms", this.getClass(), System.currentTimeMillis() - startTime);